src/Controller/Component/AuthComponent.php

```diff
@@ -537,9 +537,14 @@ public function getAuthorize($alias)
      *
      * You can use allow with either an array or a simple string.
      *
-     * `$this->Auth->allow('view');`
-     * `$this->Auth->allow(['edit', 'add']);`
-     * `$this->Auth->allow();` to allow all actions
+     * ```
+     * $this->Auth->allow('view');
+     * $this->Auth->allow(['edit', 'add']);
+     * ```
+     * or to allow all actions
```

Comment:
@ADmad no I've checked it. The only thing that's required is a new line between both fence so we can't use \`\`\`some code\`\`\` on the same line